### ISSUE TITLE
TypeError: Cannot read property 'camelCase' of undefined

### ISSUE THREAD

**Issue by manojsitapara** [open-wa/wa-automate-nodejs#1749]
I am using latest docker image file

Getting following error, when I start docker container, It was working fine, but in latest image it is not working

`TypeError: Cannot read property 'camelCase' of undefined
    at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:283:102
    at Array.map (<anonymous>)
    at Object.envArgs (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:283:60)
    at Object.cli (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:314:160)
    at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:32:68
    at Generator.next (<anonymous>)
    at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:4:12)
    at start (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:31:12)
npm notice 
npm notice New minor version of npm available! 7.13.0 -> 7.15.1
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.15.1>
npm notice Run `npm install -g npm@7.15.1` to update!
npm notice `

My old docker image has `WA Version: 3.16.0`, it is working fine.

**Comment by manojsitapara** [comment]
@smashah if possible, can you please help me to solve this issue?

**Comment by smashah** [comment]
@manojsitapara fix incoming soon

**Comment by manojsitapara** [comment]
@smashah Thank you so much Sir !!

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.1.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.1.0...4.1.1) (2021-06-04)

- build(deps): bump type-fest from 1.1.3 to 1.2.0 [`#1746`](https://github.com/open-wa/wa-automate-nodejs/pull/1746)
- build(deps): bump parse-url from 5.0.2 to 5.0.3 [`#1752`](https://github.com/open-wa/wa-automate-nodejs/pull/1752)
- build(deps-dev): bump marked from 2.0.6 to 2.0.7 [`#1745`](https://github.com/open-wa/wa-automate-nodejs/pull/1745)
- build(deps-dev): bump @types/node from 15.6.1 to 15.9.0 [`#1750`](https://github.com/open-wa/wa-automate-nodejs/pull/1750)
- build(deps): bump cfonts from 2.9.2 to 2.9.3 [`#1751`](https://github.com/open-wa/wa-automate-nodejs/pull/1751)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.25.0 to 4.26.0 [`#1741`](https://github.com/open-wa/wa-automate-nodejs/pull/1741)
- build(deps-dev): bump @typescript-eslint/parser from 4.25.0 to 4.26.0 [`#1743`](https://github.com/open-wa/wa-automate-nodejs/pull/1743)
- build(deps-dev): bump marked from 2.0.5 to 2.0.6 [`#1735`](https://github.com/open-wa/wa-automate-nodejs/pull/1735)
- Release 4.1.1 [`0903678`](https://github.com/open-wa/wa-automate-nodejs/commit/0903678c0ffb2abbe868e36783aae6037c9f025e)
- 🐛 [CLI] potential fix for #1749 [`1935cbe`](https://github.com/open-wa/wa-automate-nodejs/commit/1935cbe7e2b412bff630963322705ffd91afa731)
- 🔧 CLI: reduce json limit to 99MB [`bafaeaf`](https://github.com/open-wa/wa-automate-nodejs/commit/bafaeaf9153d17dd0809347366fc36300f02f760)
- 🐛  cli: prevent webhook error recursion [`6971130`](https://github.com/open-wa/wa-automate-nodejs/commit/6971130378366868682f803bb8f748268c08ec90)
- 🥚  patches update #1755 [`347927b`](https://github.com/open-wa/wa-automate-nodejs/commit/347927bb1f024ab5ab5619a1d62762db0771699e)

**Comment by smashah** [comment]
@manojsitapara fixed in latest version.
